src/devices/Max7219/Font.cs

```diff
@@ -0,0 +1,41 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Iot.Device.Max7219
+{
+    public class Font
```

Comment:
It's not clear to me why we have a Font class and a Fonts class as well. Do we really need both? If so, can we have better naming explaining what they represent and perhaps also adding summary comments above each class?